tests/phpunit/tests/admin/includesScreen.php

```diff
@@ -336,6 +357,10 @@ function test_help_tabs() {
 
 	/**
 	 * @ticket 19828
+	 *
+	 * @covers WP_Screen::add_help_tab
+	 * @covers WP_Screen::get_help_tab
+	 * @covers WP_Screen::remove_help_tab
```

Comment:
```suggestion
	 * @covers WP_Screen::add_help_tab
	 * @covers WP_Screen:: get_help_tab
	 * @covers WP_Screen:: get_help_tabs
	 * @covers WP_Screen::remove_help_tab
	 * @covers WP_Screen::remove_help_tabs
```

This test is testing all of the help tab(s) methods (doing too much) but adds testing the order as set by the priority.